internal/gatewayapi/testdata/backend-with-endpoint-zones.out.yaml

```diff
@@ -171,6 +179,10 @@ xdsIR:
             - host: 2.2.2.2
               port: 3001
               zone: zone2
+            metadata:
+              kind: Backend
+              name: backend-2
+              namespace: default
```

Comment:
Do we need to change this file?

Reply:
yeah, missed in https://github.com/envoyproxy/gateway/pull/6137.

Reply:
https://github.com/envoyproxy/gateway/actions/runs/15175136959/job/42673862376